app/Exceptions/DockerNotAvailableException.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Exceptions;
+
+use App\Shell\Environment;
+use App\Shell\Shell;
+use Exception;
+
+class DockerNotAvailableException extends Exception
+{
+    public function render($request = null): void
+    {
+        $console = app('console');
+        $shell = app(Shell::class);
+
+        $console->line('');
+        $console->line($shell->formatErrorMessage('Docker is not available.'));
+        $console->line('');
+
+        if (in_array(PHP_OS_FAMILY, ['Darwin', 'Linux', 'Windows'])) {
+            $osSpecificHelp = 'helpFor' . ucfirst(PHP_OS_FAMILY);
+            $this->$osSpecificHelp();
+        }
+    }
+
+    protected function helpForDarwin()
+    {
+        $console = app('console');
```

Comment:
@mattstauffer is there a way to avoid resolving this in every method?

Reply:
Yep! As with my suggestions, pass `$console` into the method call, and then accept it in each function definition